core-bundle/src/Resources/contao/languages/en/default.xlf

```diff
@@ -969,13 +969,13 @@
         <source>Edit multiple</source>
       </trans-unit>
       <trans-unit id="MSC.all.1">
-        <source>Edit multiple records at once</source>
+        <source>Edit multiple elements at once</source>
       </trans-unit>
       <trans-unit id="MSC.all_override.0">
         <source>Override multiple</source>
       </trans-unit>
       <trans-unit id="MSC.all_override.1">
-        <source>Override multiple records at once</source>
+        <source>Override multiple elements at once</source>
```

Comment:
Are you intentionally change this? Just asking because you reverted the "element" stuff. "Elements" to me is for content elements, not database records in general.

Reply:
We have agreed on using "elements" by default.

https://github.com/contao/contao/blob/8befee845bd83a1fe10f317d1e0bebcc977b6766/core-bundle/src/Resources/contao/languages/en/default.xlf#L1958-L2005